### PR TITLE
1747 adding changelog from last WD

### PR DESCRIPTION
**Pull request by mdjp**
***
<a href="https://pr-preview.s3.amazonaws.com/WebAudio/web-audio-api/pull/1748.html" title="Last updated on Sep 14, 2018, 9:57 AM GMT (5a1e1a3)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WebAudio/web-audio-api/1748/357cfcf...5a1e1a3.html" title="Last updated on Sep 14, 2018, 9:57 AM GMT (5a1e1a3)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -12031,7 +12031,22 @@ class Vec3 {
 
 <h2 id="changelog">
 Change Log</h2>
-<h3 id="changestart">
+<h3 id="changestart1">
+Since Working Draft of 19 June 2018</h3>
+* Update implementation-report.html
+* Widen range for detune values
```
Comment (rtoy) [inline index.bs]: This is actually restricting the range. Previously it was +/- inf (exclusive), the we reduced it to a much smaller range, and but then made it wider to cover more.  So compared to before, it's a restriction of the range.

File: index.bs [hunk 2]
```diff
@@ -12031,7 +12031,22 @@ class Vec3 {
...
+* PannerNode constructor throws errors
+* Rephrase algorithm for setting buffer and curve
+* Refine startRendering algorithm.
+* Spec more precisely events overlapping with SetValueCurveAtTime
```
Comment (rtoy) [inline index.bs]: "Spec more precisely events" reads better as
"Spec, more precisely, events" or "Spec events ... SetValueCurveAtTime more precisely".

Maybe we shouldn't say "Spec" but "Specify"

File: index.bs [hunk 3]
```diff
@@ -12031,7 +12031,22 @@ class Vec3 {
...
+* Add implementation report to gh-pages
+* Honor the given value in `outputChannelCount`
+* Add images for convolution doc
```
Comment (rtoy) [inline index.bs]: The convolution doc isn't part of the spec, so I'd leave this out.(If by convolution doc you mean convolution.html)

Comment (svgeesus) [reply index.bs]: Right, this is in our repo but doesn't get copied over when the spec is published

**Comment by mdjp** [comment]
Ok, so more haste less speed, I rushed this to get it in before the meeting - can you tell?

File: index.bs (hunk 3 above)
Comment (mdjp) [reply index.bs]: yes, this one crept through my filter.

File: index.bs (hunk 2 above)
Comment (mdjp) [reply index.bs]: agreed, just copying from the git logs, guess I should have read some of them a bit more closely

**Pull request merged**